meta-resin-common/recipes-support/resin-init/resin-init-flasher/dnsmasq.conf.systemd

```diff
@@ -0,0 +1,8 @@
+[Unit]
+Wants=resin-net-config.service
+After=resin-net-config.service
```

Comment:
Yes, see the resinOS issue for this and discussion there. It has been discussed already